### DISCUSSION TITLE
Folder search cannot find anything

### DISCUSSION THREAD

**Discussion by joetin** [photoprism/photoprism#1875] [Q&A] [answered]
Happy new year!

I used Build 211018-e200f322 and anything is fine.
Several days ago, I updated to newest Build 211215-93b26f19 and found I cannot get any results in the folder search.
I also tested in the demo site.

There is a folder named "German Film Award 2012" 
and I searched "film", nothing shown. 
"award", nothing shown. 
"2012", nothing shown.

And there is another folder located in "/2018/07" 
I searched "2018", shown.
"07", nothing shown.

I don't know how to fix it.
Could anyone tell me how to solve this?
Thank you.

**Comment by graciousgrey** [comment, marked as answer]
The search only looks for exact matches unless you are using wildcards:

- `German Film Award 2012` --> works
- `*film*` --> works
- `*2012` --> works
- `*07` --> works

**Reply by joetin** [reply]
it really helps. Thank you.
In previous versions, it also worked without wildcards so that I didn't try wildcards.
